website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/service-catalog-config.mdx

```diff
@@ -169,12 +169,27 @@ updated whenever changes are merged to the associated branch in that repository.
 
 ## Execution Mode
 
-If using v2.2.0 or above, the Service Catalog app allows you to set an [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode) for your Terraform workspaces. There are three modes to choose from:
+If using v2.2.0 or above, the Service Catalog app allows you to set an [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode) for your Terraform workspaces. There are two modes to choose from:
 
  - The default value is "Remote", which instructs Terraform Cloud to perform runs on its disposable virtual machines.
- - Switching to "Local" mode allows Terraform to execute runs locally using the CLI workflow. 
  - Selecting "Agent" mode allows you to run Terraform operations on isolated, private, or on-premises infrastructure. This option requires you to create an Agent Pool in your organization beforehand, then provide that Agent Pool's id when you order a new workspace through the Service Catalog.
 
 <!-- BEGIN: TFC:only name:pnp-callout -->
 @include 'tfc-package-callouts/agents.mdx'
-<!-- END: TFC:only name:pnp-callout -->
\ No newline at end of file
+<!-- END: TFC:only name:pnp-callout -->
+
+## Workspace Name
+
+Version 2.4.0 of the Service Catalog for Terraform introduces the ability to set custom names for your Terraform workspaces. Previously, workspace names defaulted to the ServiceNow RITM numbers without an option to change them. Now, you can choose a custom name that the RITM number will automatically append to, such as: `prod-us-east-1_RITM0010231`. If you leave the name blank, it will continue to default to the RITM number.
+
+Workspace names can include letters, numbers, dashes (`-`), and underscores (`_`), and should not exceed 90 characters.
```

Comment:
Small nit, when we say "should not exceed 90 characters", can they technically be longer? Or is 90 the maximum length?

Reply:
Yes, workspace names can be very long (I tried)! They just don't look good in the UI. The 90 char recommendation comes from the TFC workspace documentation page.